tls/README.md

```diff
@@ -1302,7 +1304,48 @@ To generate your own cert text, see the [DER to C script](https://github.com/wol
 
 <br />
 
+## <a name="ech">Encrypted Client Hello</a>
+
+Encrypted Client Hello (ECH) encrypts sensitive fields in the client hello step of the TLS handshake. The client-ech example connects to a cloudflare server that is setup to test different TLS options including ECH. To build wolfSSL for this ech example run `./configure --enable-ech && make && sudo make install`.
+
+This test is successful if the cloudflare http response shows that `sni=encrypted`.
```

Comment:
```
./configure --enable-ech && make && sudo make install
```

```
% ./client-ech
-1 -188
```

Reply:
Hello there I was trying to run this example and I came upon the same error. I firstly changed the intermediate certificate that is now expired with the current one ( i could try to comment here the changes) but the error persisted.

Any idea what could cause this error? Is this example going to be merged or it is now inactive?

Thank you.

Reply:
Update: The **-188** Error code refers to the inability of wolfSSL to verify the certificate chain because it lacks the proper CA certificate. After some search i found out that in the `certs/ech-client-cert.pem` file, the Root CA is missing, thus being impossible to verify the whole chain. Maybe in some configuration it can be loaded from the system but that was not the case for me. Adding the Root CA in the bottom of the `certs/ech-client-cert.pem` file solved the issue. The connection was established and the `sni=encrypted` was displayed.

Reply:
Thanks for the heads up, I've added the correct cert chain.